packages/arb-token-bridge-ui/src/components/TransferPanel/NetworkListbox.tsx

```diff
@@ -70,7 +70,7 @@ export function NetworkListbox({
                         size={20}
                       />
                     </div>
-                    <span className="max-w-[140px] truncate">
+                    <span className="max-w-[150px] truncate">
```

Comment:
Expanded a bit, otherwise the name is truncated

Reply:
I think, design-wise, we want the box to fit the content so `min-w` may not be ideal here